src/keycloak/chart/templates/statefulset.yaml

```diff
@@ -132,6 +132,11 @@ spec:
           {{- if .Values.env }}
             {{- tpl (toYaml .Values.env) . | nindent 12 }}
           {{- end }}
+            {{- if or (contains "rfcurated" .Values.image.repository) (contains "rfcurated" (toString .Values.image.tag)) }}
+            # Ensure javaagent is disabled (only for rfcurated unicorn flavor)
+            - name: JDK_JAVA_OPTIONS
+              value: "--add-exports=java.base/sun.security.internal.spec=ALL-UNNAMED --add-exports=java.base/sun.security.provider=ALL-UNNAMED"
+            {{- end }}
```

Comment:
I think this is sane, but just want to call out this will potentially clobber `JDK_JAVA_OPTIONS` if someone was setting that via `env`? Also unsure if we'd need to maintain this over time to keep it in sync with the RF image upstream (just minus the provider).

Reply:
Resolving - discussed offline that this is an advanced java env that is not likely to be used by end users, and unlikely to need much maintenance.